docs/ota-client-guide/modules/ROOT/pages/client-provisioning-methods.adoc

```diff
@@ -14,67 +14,80 @@ endif::[]
+
+In this method, the following steps occur:
+
+. You download your `credentials.zip` with the provisioning key inside.
+. You bitbake an image with the default provisioning method configured, providing bitbake with your `credentials.zip`.
+. You flash that image on as many devices as you wish.
+. The first time each device comes online, it connects to a special endpoint on the device gateway and says, “Hi, I’m a new device, and I don’t have a TLS certificate yet. Here’s my provisioning key.”
+. Our crypto service generates a new keypair and X.509 certificate for the device, signs that certificate with the private Fleet Root CA in the Vault instance, and sends the whole bundle to the vehicle.
+. The crypto service deletes the vehicle’s private key and the device stores its new keypair and certificate.
+
+This entire transaction is secured with TLS, using the pinned device gateway certificate provisioned in the image.
+
+.Summary of Shared credential provisioning
+[caption="Figure 1: "]
+image::shared-cred-provisioning.png[1280, 500]
 
-This method is fine for provisioning devices quickly but if a malicious actor steals your provisioning key, there's no way to prevent unauthorized devices from provisioning. You'd have to blacklist the provisioning key for all devices and issue a new one.
 
 == Provisioning with device credentials
 
-If you're using OTA Connect in production, you should provision with device credentials.
-In this scenario, the OTA Connect server doesn't issue any credentials to devices. You need to preinstall the device certificates yourself.
-How you get the certificates on to your devices is your decision. Your chosen method can depend on many variables such as the storage method on the device, whether you have connectivity at your factory, and whether you choose an internal or third party root of trust for your certificates.
-For example, you might have a deployment process where your build system requests a certificate from your CA while a software image is being built. The device certificate is then installed on the image just before it is flashed to the target device.
-Once a device boots up, it connects to the OTA Connect server and provides the pre-installed device certificate to verify the device identity. The OTA Connect server then verifies that the certificate came from your CA.
-With this method, it is extremely difficult for an unauthorized device to join your fleet.
+We usually recommend this provisioning method if you have high cybersecurity compliance needs, and especially if you have devices with a hardware security module. In this method there is no shared credential. Instead, the following steps occur:
 
+. You provide us with your own Fleet Root CA’s certificate (but NOT the private key).
+. Then, you make sure that each device acquires, in some out-of-band process, an X.509 certificate signed by your fleet root CA.
```

Comment:
In all other instances, "fleet root CA" was capitalized. Do you care?

Also, do you want to be consistent about boldface vs. all-caps for emphasis? I'm just asking about "NOT". I don't have a strong opinion here, just being pedantic.

Reply:
@patrickvacek I corrected this in one of the commits. Maybe it was somehow lost after @tkfu rebased. We can change this later if you like. I would like it to be capitalized to maintain consistency.